messages/de/app.php

```diff
@@ -167,17 +167,17 @@
     'User' => 'Benutzer',
     'Users' => 'Benutzer',
     'View' => 'Ansehen', // view server button on Dashboard
-    'Want to be a partner?' => 'Du möchtest Partner werden?',
+    'Want to be a partner?' => 'Möchtest du Partner werden?',
```

Comment:
Your translation suggests that the website is inviting the user to become a partner, and while I can see how that could be seen that way from the source string, it's asking the user if they already have the feeling they want to become a partner.

Reply:
Du möchtest  partner werden => Satz

Fragen haben eine andere Satzstellung als Sätze

Reply:
“Du möchtest Partner werden?” is correct German. It's a [yes-no question](https://de.wikipedia.org/wiki/Entscheidungsfrage), which can appear in a [verb-second word order](https://de.wikipedia.org/wiki/V2-Stellung).